indices/apis.js

```diff
@@ -117,7 +108,7 @@ function getRecords () {
     ]
 
     return Object.assign(
-      { objectID: record.url.replace('https://electronjs.org/docs/api/', 'api-') },
+      { objectID: record.url.replace('https://electronjs.org/docs/api/', 'api-') + `-${record.slug}` },
```

Comment:
It would help if I actually hit the save button :facepalm:

For consistency:
```suggestion
      { objectID: `${record.url.replace('https://electronjs.org/docs/api/', 'api-')}-${record.slug}` },
```

Reply:
Seems reasonable, accepted the suggestion.